cpp/custom-dataset/README.md

```diff
@@ -0,0 +1,52 @@
+# Custom Dataset Example with the PyTorch C++ Frontend
+
+This folder contains an example of loading a custom image dataset with OpenCV and training a model to label images, using the PyTorch C++ frontend.
```

Comment:
Is there a way to provide instructions for installing OpenCV correctly? There seems to be multiple ways to do it, and having one that's guaranteed to work with this example would be beneficial. (We can have a separate README to document this.)

Reply:
I will write a separate README for it.

Reply:
@yf225 I have always installed OpenCV from package manager on Arch and Fedora for normal usage. It is hard to write instructions that work every where. Even the official documentation of OpenCV focuses on building from source. I did what I could. Please see if this is enough.